### PR TITLE
added note about docker and go dependency to the readme

### PR DESCRIPTION
**Pull request by paulbellamy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -217,6 +217,10 @@ kubectl create -f scope-probe-ds.yaml
 
 ## <a name="developing"></a>Developing
 
+Building scope from source depends on the latest versions of
```
Comment (2opremio) [inline README.md]: Maybe capitalize Scope?

**Comment by 2opremio** [comment]
Minor-comment, LGTM.

File: README.md [hunk 2]
```diff
@@ -217,6 +217,10 @@ kubectl create -f scope-probe-ds.yaml
...
+[docker](https://www.docker.com/) and [go](https://golang.org/), so
```
Comment (2opremio) [inline README.md]: Actually, does it really require Go? The build it's containerized these days.

Comment (paulbellamy) [inline README.md]: Not entirely. `make deps` does `go get github.com/mjibson/esc` which is used to build `prog/static.go` (which is not containerized).

**Comment by paulbellamy** [comment]
removed dependency on go on the host, and clarified the relationship between `make` and `make deps` a bit.

@davkal does this still work for you?

**Comment by davkal** [comment]
Still works, @paulbellamy thx

File: README.md [hunk 3]
```diff
@@ -217,14 +217,16 @@ kubectl create -f scope-probe-ds.yaml
 
 ## <a name="developing"></a>Developing
 
-The build is in two stages. `make deps` installs some tools we use later in
-the build. `make` builds the UI build container, builds the UI in said
-container, builds the backend build container, builds the app and probe in a
-said container, and finally pushes the lot into a Docker image called
-**weaveworks/scope**.
+Building Scope from source depends on the latest version of
```
Comment (2opremio) [inline README.md]: Does it really need to be the latest one?

Comment (paulbellamy) [inline README.md]: Probably not but scope doesn't really target anything other than the latest docker, so may as well have them upgrade it.

**Pull request merged**